avocado/core/remote/runner.py

```diff
@@ -325,8 +329,8 @@ class VMTestRunner(RemoteTestRunner):
     Test runner to run tests using libvirt domain
     """
 
-    def __init__(self, job, result_proxy):
-        super(VMTestRunner, self).__init__(job, result_proxy)
+    def __init__(self, job, result_proxy, result):
```

Comment:
the next patch does not make the use of it, so right now I see it as not really useful change. I'd recommend either sending this patch in the patchset with that change, or even squashing it into that patch. Right now it looks contra-productive to have results_proxy and single results together. It'd be actually better to always inject the `Result` to result_proxy, but that already happens when there is no result plugin enabled...